apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -817,17 +825,35 @@ class ScriptEditor extends React.Component {
           </label>
           <div>
             <h4>Teacher Resources</h4>
-            <div>
-              Select the Teacher Resources buttons you'd like to have show up on
-              the top of the script overview page
-            </div>
-            <ResourcesEditor
-              inputStyle={styles.input}
-              resources={this.state.teacherResources}
-              updateTeacherResources={teacherResources =>
-                this.setState({teacherResources})
-              }
-            />
+            {!this.props.isMigrated && (
+              <div>
+                <div>
+                  Select the Teacher Resources buttons you'd like to have show
+                  up on the top of the script overview page
+                </div>
+                <ResourcesEditor
+                  inputStyle={styles.input}
+                  resources={this.state.teacherResources}
+                  updateTeacherResources={teacherResources =>
+                    this.setState({teacherResources})
+                  }
+                />
+              </div>
+            )}
+            {this.props.isMigrated && (
```

Comment:
It looks like maybe the code for deciding which editor to show and the parts of the editor might be the same between Course and Script edit? Is it worth creating a component to share?

Reply:
oh good thought -- I could actually roll this logic into `ResourcesEditor` and have that handle figuring out what to show. That would have the added benefit of not having to preview the dropdown here too

Reply:
Done in 1caab4299b9. I think it consolidates some of this logic but let me know if you disagree.

The only downside is that the editor looks a bit cluttered on migrated scripts but we can clean this up later
<img width="1096" alt="Screen Shot 2021-03-31 at 10 35 21 AM" src="https://user-images.githubusercontent.com/46464143/113186804-f4d82180-920c-11eb-9a66-3713f1e66427.png">